test/parallel/test-vm-module-basic.js

```diff
@@ -124,12 +124,24 @@ const util = require('util');
 // Check to throws invalid exportNames
 {
   assert.throws(() => new SyntheticModule(undefined, () => {}, {}), {
-    message: 'The "exportNames" argument must be an Array of strings.' +
-      ' Received undefined',
+    message: 'The "exportNames" argument must be an ' +
+        'Array of unique strings.' +
+        ' Received undefined',
     name: 'TypeError'
   });
 }
 
+// Check to throws duplicated exportNames
+// https://github.com/nodejs/node/issues/32806
+{
+  assert.throws(() => new SyntheticModule(undefined, () => {}, {}), {
+    message: 'The "exportNames" argument must be an ' +
+        'Array of unique strings.' +
+        ' Received undefined',
+    name: 'TypeError',
+  });
+}
+
```

Comment:
can you add a test for explicitly passing duplicate names?

Reply:
oh, no. I forget to change the first parameter😟